### PR TITLE
APP-373: Disable shake in Dev and Production

### PR DESCRIPTION
**Pull request by hedvigoscar**
- Only add Shake in staging builds
- Bonus: Use the correct AGP version in `buildSrc`

### Checklist

- [ ] Functionality is covered by an integration test
- [ ] Functionality is accessible in engineering mode

### CODE REVIEW

File: app/src/engineering/java/com/hedvig/app/ShakeInitializer.kt [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+class ShakeInitializer : Initializer<Unit> {
+    override fun create(context: Context) {
+        // Only initialize Shake when running a staging build
+        if (BuildConfig.APPLICATION_ID != "com.hedvig.test.app") {
+            return
+        }
+        Shake.getReportConfiguration().isInvokeShakeOnShakeDeviceEvent = true
+        Shake.getReportConfiguration().isInvokeShakeOnScreenshot = true
+        Shake.getReportConfiguration().isShowFloatingReportButton = true
```
Comment (hugokallstrom) [inline app/src/engineering/java/com/hedvig/app/ShakeInitializer.kt:16]: Will these be added in another PR?

**Review by hugokallstrom — COMMENTED** [review]
Very cool

File: app/src/engineering/java/com/hedvig/app/ShakeInitializer.kt (hunk 1 above)
Comment (hedvigoscar) [reply app/src/engineering/java/com/hedvig/app/ShakeInitializer.kt:16]: 🤦 sorry
